core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -627,4 +627,11 @@ private T thisT() {
     T thisT = (T) this;
     return thisT;
   }
+
+  /**
+   * Returns the internal ExecutorPool for offloading tasks.
+   */
+  protected ObjectPool<? extends Executor> getExecutor() {
+    return this.executorPool;
```

Comment:
Should you be using `this.offloadExecutorPool` if the purpose it to use it for "some time-consuming tasks"? I think the `offloadExecutorPool` is used for resolver tasks but seems like could be used here as well. But both are initialized to `DEFAULT_EXECUTOR_POOL` BTW so for default case there's no difference.